client/my-sites/domains/components/form/country-select.jsx

```diff
@@ -52,7 +52,7 @@ const CountrySelect = createReactClass( {
 			options.push( {
 				key: 'loading',
 				label: this.props.translate( 'Loading…' ),
-				disabled: 'disabled',
+				disabled: false,
```

Comment:
`disabled="disabled"` sounds more like `disabled: true` to me, so I'm not sure why `disabled: false` here...

Reply:
You read it correctly. The idea is to enable this option instead of disabling it as otherwise this placeholder will not be displayed.

Reply:
so why not just drop `disabled` ?

Reply:
That's a very good suggestion indeed 😅.